src/services/upload/uploadManager.ts

```diff
@@ -323,22 +339,10 @@ class UploadManager {
                     .get(file.collectionID)
                     .push(file);
             }
-            if (fileUploadResult === FileUploadResults.FAILED) {
-                this.failedFiles.push(fileWithCollection);
-                setData(LS_KEYS.FAILED_UPLOADS, {
```

Comment:
Just to confirm, we don't need to track such failures now? What was the use-case for adding this in the first place?